client/components/phone-input/index.jsx

```diff
@@ -16,21 +18,22 @@ import { countries } from './data';
 const PhoneInput = React.createClass( {
 	propTypes: {
 		onChange: React.PropTypes.func,
-		initialValue: React.PropTypes.string,
-		selectedCountryCode: React.PropTypes.string.isRequired,
+		value: React.PropTypes.string,
+		initialCountryCode: React.PropTypes.string,
 		countriesList: React.PropTypes.object.isRequired
 	},
 
 	getDefaultProps() {
 		return {
-			initialValue: '',
+			value: '',
 			onChange: noop,
+			initialCountryCode: 'us'
 		};
 	},
 
 	getInitialState() {
-		const selectedCountry = countries[ this.props.selectedCountryCode ];
-		const formattedNumber = formatNumber( this.props.initialValue, selectedCountry );
+		const selectedCountry = countries[ this.props.initialCountryCode ];
```

Comment:
The problem here is:
I have a stored value: +382 67 112233
When I open the form, it turns it into: +1 38267112233

Maybe I didn't use it properly. The idea was, if stored, try to parse country. If cannot, use default. But maybe I'm missing something?